packages/hooks/src/use-device-screen.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+
+export function useDeviceScreen(): {
```

Comment:
I think we can change this to a generic hook like `useWindowSize` 
isMobile and isTable can be different depend on each usecase. just return with and height of the screen, the consumer will decide how to use it.